<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tuneNewDialog</class>
 <widget class="QDialog" name="tuneNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="tuneName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>57</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true">File</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>34</y>
     <width>57</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Division</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>RH Channel</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRH">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>128</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>LH Channel</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLH">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>128</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tuneNewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tuneNewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
